styles/style.css

```diff
@@ -0,0 +1,333 @@
+nav {
+  opacity: 1;
+  background-color:#F4F4F4;
+  position: sticky;
+  top: 0;
+  z-index: 1;
+  border-bottom: solid black;
+  border-bottom-width: .05em;
+}
+
+nav ul {
+  list-style: none;
+  padding: 30px;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
```

Comment:
Good use of grid! To clean it up just a little, you can do grid-template-columns: repeat(4,  1fr) if ya want.